README.md

```diff
@@ -27,14 +27,19 @@ Commit your code regularly and meaningfully. This helps both you (in case you ev
 Be prepared to demonstrate your understanding of this week's concepts by answering questions on the following topics. You might prepare by writing down your own answers before hand.
 
 1. What is the DOM?
+   Dom or Document Object Model is a model or a blueprint of how the website should look like. In Dom, we can write Javascript to create, modify and delete HTML elements, set style, classes and attributes; and listen and respond to event to make website interactive.
```

Comment:
The DOM is an object representation of the HTML that we can manipulate using Javascript

we can write Javascript to create, modify and delete HTML elements *on the DOM*